config/prod/ahayden-test-awslinux.yaml

```diff
@@ -0,0 +1,46 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2.yaml
```

Comment:
there's a new template version, "managed-ec2-v9.yaml", please use that one: https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/3/files